drivers/volume/kdmp/kdmp.go

```diff
@@ -617,6 +617,24 @@ func (k *kdmp) StartRestore(
 		volumeInfo.SourceVolume = bkpvInfo.Volume
 		volumeInfo.DriverName = storkvolume.KDMPDriverName
 
+		// If restore replace policy is retain , then skip the pvc
+		if restore.Spec.ReplacePolicy == storkapi.ApplicationRestoreReplacePolicyRetain {
+			pvc, err := core.Instance().GetPersistentVolumeClaim(pvc.Name, restoreNamespace)
+			if err != nil && !k8serror.IsNotFound(err) {
+				logrus.Errorf("failed to get pvc object for %s/%s: %v", restoreNamespace, pvc.Name, err)
+				volumeInfo.Status = storkapi.ApplicationRestoreStatusFailed
+				volumeInfos = append(volumeInfos, volumeInfo)
+				continue
+			}
+			if err == nil {
+				logrus.Debugf("Skipping pvc %s/%s as the replace policy is set as %s", restoreNamespace, pvc.Name, storkapi.ApplicationRestoreReplacePolicyRetain)
+				volumeInfo.Status = storkapi.ApplicationRestoreStatusSuccessful
+				volumeInfo.RestoreVolume = pvc.Name
```

Comment:
Suggestion: Can we update the reason field saying this PVC was skipped restoring that way we know what PVC is restored or not and would be useful for debugging?

Reply:
done.